_examples/prop.go

```diff
@@ -34,7 +34,7 @@ func main() {
 		panic(err)
 	}
 	if reply != dbus.RequestNameReplyPrimaryOwner {
-		fmt.Fprintln(os.Stderr, "name already taken")
+		_, _ = fmt.Fprintln(os.Stderr, "name already taken")
```

Comment:
Why do this?

Reply:
To eliminate IDE warnings.

Reply:
Sigh, I'm not really a fan of changing code because of an overzealous IDE. The intent behind this code is that the error doesn't matter and the way it currently is is fine. That said, I'll merge this anyway as I don't work on the prop stuff and you do and it bugged you enough to open the PR.

Thanks!